src/ua_types.c

```diff
@@ -1437,6 +1438,7 @@ static void nopClear(void *p, const UA_DataType *type) { }
 
 const
 UA_clearSignature clearJumpTable[UA_DATATYPEKINDS] = {
+    NULL,
```

Comment:
The same here.

also for binary, JSON and xml encoding.

Reply:
good point, for now I put a NULL to compile and trying to test the happy path as a first step.